app/controllers/home_controller.rb

```diff
@@ -13,13 +13,10 @@ def index
   end
 
   def landing
-    recipe_images = RecipeImage.find([549, 376, 529, 526])
```

Comment:
ah. i just saw you comment @dpick 
i would suggest to add a featured flag for recipes and fetch these for the landing page. what do you think? I can add it to the PR